yarn-project/archiver/src/archiver/config.ts

```diff
@@ -74,6 +74,7 @@ export function getConfigEnvVars(): ArchiverConfig {
       ? EthAddress.fromString(CONTRACT_DEPLOYMENT_EMITTER_ADDRESS)
       : EthAddress.ZERO,
     decoderHelperAddress: EthAddress.ZERO,
+    headerDecoderHelperAddress: EthAddress.ZERO,
```

Comment:
Having test specific things in here makes my skin itch 😬 Was this mainly for the test, or is it used somewhere. Same for the `decoderHelperAddress`.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4015/commits/388aa21bf22c876671f98a2868fca63d23a16a05